main.js

```diff
@@ -1,51 +1,46 @@
 $('document').ready(function(){
 
-  $('#button').on('click', requestUrl2Png("PBB2128AF2E314E", "SecretToken"));
+  $('#leButton').on('click', saveCookie);
+  $('#button').on('click', requestUrl2Png());
 
 });
 
-function requestUrl2Png(api, api_secret){
-
-  var userWebsite = getCookie('websiteUrl') 
-  var options = {
-    url: userWebsite,
-    fullpage: true,
-    protocol: 'http',
-    ttl: 4000000
-  }
-  var queryString = $.param(options)
-  var token = CryptoJS.MD5(queryString + api_secret).toString();
-  var callUrl = 'https://api.url2png.com/v6/' + api + '/' + token +'/png/?' + queryString
-
-  $.ajax({url: callUrl,
-          success: function(result){
-                      console.log("success")
-                    },
-          xhrFields: {
-            withCredentials: true,
-            'Allow-Control-Access-Origin': true
-          }
-  });
+function requestUrl2Png(){
+  var api = "PBB2128AF2E314E"
+      api_secret = "secretKey"
+
+      userWebsite = getCookie(cookieName) 
+      options = {
+          url: userWebsite,
+          fullpage: true,
+          protocol: 'http',
+          ttl: 4000000
+        }
+      queryString = $.param(options)
+      token = CryptoJS.MD5(queryString + api_secret).toString();
+      callUrl = 'https://api.url2png.com/v6/' + api + '/' + token +'/png/?' + queryString
+
+  $('#image').prepend('<img style="display: inline-block; width: 90%; margin: 0 auto" src="' + callUrl + '" />')
```

Comment:
Embarrassingly enough. I figured that we were originally generating a URL to an image. and it just needed to be embedded into the document. Lucky us!